bndtools.core/src/bndtools/launch/util/LaunchUtils.java

```diff
@@ -19,10 +19,7 @@
     private LaunchUtils() {}
 
     public static IResource getTargetResource(ILaunchConfiguration configuration) throws CoreException {
-        String target = configuration.getAttribute(LaunchConstants.ATTR_LAUNCH_TARGET, (String) null);
-        if (target == null || target.length() == 0) {
-            return null;
-        }
+        String target = getTargetName(configuration);
 
         IResource targetResource = ResourcesPlugin.getWorkspace().getRoot().findMember(target);
```

Comment:
Is it OK to possible pass `null` to `findMember` here? You removed the null check that returns, so null can now be passed to `findMember`.